openedx_events/enterprise/data.py

```diff
@@ -22,3 +25,62 @@ class SubsidyRedemption:
     subsidy_identifier = attr.ib(type=str)
     content_key = attr.ib(type=str)
     lms_user_id = attr.ib(type=int)
+
+
+@attr.s(frozen=True)
+class LedgerTransactionReversal:
+    """
+    Attributes of an ``openedx_ledger.Reversal`` record.
+
+    Arguments:
+        uuid (str): Primary identifier of the record.
+        created (datetime): When the record was created.
+        modified (datetime): When the record was last modified.
+        idempotency_key (str): Client-generated unique value to achieve idempotency of operations.
+        quantity (int): How many units of value this reversal represents (e.g. USD cents).
+        state (str): Current lifecyle state of the record, one of (created, pending, committed, failed).
+    """
+
+    uuid = attr.ib(type=UUID)
+    created = attr.ib(type=datetime)
+    modified = attr.ib(type=datetime)
+    idempotency_key = attr.ib(type=str)
+    quantity = attr.ib(type=int)
+    state = attr.ib(type=str)
+
+
+@attr.s(frozen=True)
+class LedgerTransaction:
+    """
+    Attributes of an ``openedx_ledger.Transaction`` record.
```

Comment:
I think I need more context on the events to review these new data classes. Could you include some docs in the PR?

Reply:
https://github.com/openedx/openedx-ledger/blob/180ace9975703e080803bd3a0e905f76b4e9f8c6/openedx_ledger/models.py#L304

I can update the code with links to same.

Reply:
Yes! I think we need better descriptions and documentation in case this is adopted elsewhere.